ClothesApp/WebAPI/Controllers/ProductsController.cs

```diff
@@ -0,0 +1,101 @@
+using Application.Dtos.Products;
+using Application.Dtos.Reviews;
+using Application.Interfaces.Services;
+using Domain.Entities;
+using Microsoft.AspNetCore.Mvc;
+
+namespace WebAPI.Controllers;
+
+[ApiController]
+[Route("api/products")] 
+public class ProductsController : ControllerBase
+{
+    private readonly IProductService _productService;
+
+    private readonly IReviewService _reviewService;
+
+    public ProductsController(IProductService productService, IReviewService reviewService)
+    {
+        _productService = productService;
+        _reviewService = reviewService;
+    }
+    
+    [HttpGet]
+    [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(IList<ProductDto>))]
+    public async Task<ActionResult> GetProducts()
```

Comment:
Why would you get all possible products without any filtering? This might be very heavy query